### PR TITLE
bestblock: move from graphql to postgresql [Marek]

### PR DESCRIPTION
**Pull request by nicarq**
`Moving the bestblock query out of graphql to save some cpu resources.`  from #113 

rebased + remove graphql from other endpoints

We still need to manually create a `View` that under other circumtances is created by Hasura.

```

CREATE VIEW "TransactionOutput" AS  SELECT tx_out.address,
    tx_out.value,
    tx.hash AS "txHash",
    tx_out.index
   FROM tx
     JOIN tx_out ON tx.id = tx_out.tx_id;
```

(this is from another call that we make. I will optimize those later on)

### CODE REVIEW

**Comment by nicarq** [comment]
Tested it

```
{
  "epoch": 238,
  "slot": 106704,
  "hash": "64ac0b138fe325019f21fcf47a54da3dbe9da9b24e3ee181d99e93a0cfb5d38c",
  "height": 5134082
}
```

on production and also using this PR (no graphql running at all!

**Comment by nicarq** [comment]
cc @mmahut

**Comment by nicarq** [comment]
(still missing an endpoint. will try to migrate)

**Comment by mmahut** [comment]
Does this mean `utxoSumForAddresses` is not used currently?

**Comment by nicarq** [comment]
@mmahut yeah. I asked Seba and he confirmed

File: src/services/transactionHistory.ts [hunk 1]
```diff
@@ -267,8 +265,10 @@ export const askTransactionHistory = async (
     txIndex: number,
   }
   , untilNum: number
-) : Promise<UtilEither<TransactionFrag[]>> => {
-  const addressTypes = getAddressesByType(addresses);
+): Promise<UtilEither<TransactionFrag[]>> => {
+    const addressTypes = getAddressesByType(addresses);
+    console.log("History query")
+    console.log(askTransactionSqlQuery)
```
Comment (nicarq) [inline src/services/transactionHistory.ts:271]: need to delete this. I missed it

**Review by ashisherc — APPROVED** [review]

File: src/services/utxoSumForAddress.ts [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+  //   JSON.stringify({ "query": query, "variables": {"addresses":addresses} }),
+  //   contentTypeHeaders);
+  // if(   "data" in ret && "data" in ret.data
+  //      && "utxos_aggregate" in ret.data.data
+  //      && "aggregate" in ret.data.data.utxos_aggregate
+  //      && "sum" in ret.data.data.utxos_aggregate.aggregate
+  //      && "value" in ret.data.data.utxos_aggregate.aggregate.sum)
+  //   return { kind: "ok", value: ret.data.data.utxos_aggregate.aggregate.sum.value };
+  // else
+    return { kind: "error", errMsg: "utxoSumforAddresses endpoint not available."};
```
Comment (SebastienGllmt) [inline src/services/utxoSumForAddress.ts:30]: Although we don't use it, can we make sure no other service is querying our endpoint for this?

**Review by SebastienGllmt — APPROVED** [review]

File: src/services/utxoSumForAddress.ts (hunk 2 above)
Comment (SebastienGllmt) [reply src/services/utxoSumForAddress.ts:30]: Update: according to Marek, this is used by some 3rd party

Comment (nicarq) [reply src/services/utxoSumForAddress.ts]: @SebastienGllmt @mmahut endpoint added

**Comment by mmahut** [comment]
This has been tested and is working fine.

**Pull request merged**